src/esp/gfx/LightSetup.h

```diff
@@ -25,8 +25,8 @@ enum class LightPositionModel {
 
 /** @brief Contains a single light's information. */
 struct LightInfo {
-  Magnum::Vector3 position;
-  Magnum::Color4 color{1};
+  Magnum::Vector4 position;  // use w == 1 for position or w == 0 for direction
```

Comment:
Then can we call it some other names, e.g., `lightVector` instead of `position`? 
It is less obvious, but that is the point.
User does not know what is a `lightVector`, and she will have to go to check the definition and usage.

Reply:
sounds good to me :+1:

Reply:
I'd say *technically* it's still a position, just that it's in [homogeneous coordinates](https://en.wikipedia.org/wiki/Homogeneous_coordinates) :wink:

Reply:
I still believe `position` somehow is misleading from the users' perspective, and I would suggest renaming it.

Reply:
I agree with @bigbike here. There's not much value in being *technically* right if it confuses our users. I don't want to try to teach our users homogeneous coordinates.